covalent_dispatcher/_cli/service.py

```diff
@@ -37,13 +37,21 @@
 from distributed.core import connect, rpc
 
 from covalent._data_store.datastore import DataStore
-from covalent._shared_files.config import ConfigManager, get_config, set_config
+from covalent._shared_files.config import (
+    ClientConfigManager,
+    CMType,
+    ServerConfigManager,
+    get_config,
+    set_config,
+)
 from covalent.utils.migrate import migrate_pickled_result_object
 
-cm = ConfigManager()
+# cm = ConfigManager()
```

Comment:
looks like you left a stray comment here

Reply:
fixed